news/2022-10-04-changes-to-osu-mania-sr-and-pp.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "Changes to osu!mania Star Rating & Performance Points"
+date: 2022-10-04 12:00:00 +0000
+---
+
+Big changes are arriving in the osu!mania scene as difficulty calculation receives its first set of updates in a long time! Read on for more info!
+
+![](/wiki/shared/news/banners/star-rating.jpg)
+
+As these new changes are the first to be made in nearly [4 years](https://osu.ppy.sh/home/news/2018-05-16-performance-updates) for performance points and a whopping 8 years for star rating, it may be helpful to check the respective wiki pages for [star rating](/wiki/Beatmap/Star_rating) and [performance points](/wiki/Performance_points) to get a quick run-down on how everything works.
+
+Let's dive straight into what's new:
+
+## osu!mania changes
+
+### Long note difficulty changes
+
+A [new change](https://github.com/ppy/osu/pull/17913) in long note calculation fixes issues with fast releases giving disproportionately more star rating than they should.
+
+Before this change, all overlapping long notes would receive a fixed bonus. After this change, this bonus now scales proportionally with how long ago the last long note release was.
+
+This change also fixes an oversight with long notes in the rightmost column of a map taking precedence in overlap calculation.
+
+An [interactive demonstration](https://www.desmos.com/calculator/7eboanfzne) by [**Evening**](https://osu.ppy.sh/users/2193881) helps visualize how this change affects the long note bonus if you're interested.
```

Comment:
Mentioned in the dev server that for those seeking for extra information, https://eve-ning.github.io/#/blog/004/ could be linked as it goes much more in depth on how the algorithm has changed.

Helpful annex given that it's the __only__ place where this information is compiled.

Small tangent however: this solution isn't fool proof as the release timings were accounted as if they were initial press timers, which I'm unsure whether it be good to mention or just fix moving on.

Reply:
added a link to the blog post